orbstation/antagonists/traitor/objectives/final_objective/malware_injection.dm

```diff
@@ -0,0 +1,417 @@
+#define APCS_REQUIRED_FOR_MALF 5
+#define INJECTOR_HACK_TIME 20 SECONDS
+
+/datum/traitor_objective/final/malware_injection
+	name = "Hack %NUMBER% APCs, and inject the AI with malware in the %DEPT%"
+	description = "Press the button to call down a drop pod containing a malware injector and a circuit board for a malware injection console. \
+	Use the injector to hack %NUMBER% APCs, and then go to the %DEPT% to build the console. Use the injector on the console to begin \
+	corrupting the AI. WARNING: This process will alert the AI and trigger nearby alarms."
+
+	///Number of APCs that need to be hacked in order to use the injector on the console.
+	var/apcs_required = APCS_REQUIRED_FOR_MALF
```

Comment:
Am I wrong in thinking that this is never actually applied?
It seems like you could just malfunction the AI as soon as you build the console, though it will not get any APC points.

Reply:
Oh, yeah, this is a relic from when I started coding the objective and hadn't made it a define yet